src/ansys/fluent/core/services/field_data.py

```diff
@@ -61,9 +61,14 @@
 )
 from ansys.fluent.core.services.streaming import StreamingService
 from ansys.fluent.core.utils.deprecate import deprecate_argument, deprecate_arguments
+from ansys.fluent.core.variable_strategies import (
+    FluentFieldDataNamingStrategy as naming_strategy,
+)
 
 logger = logging.getLogger("pyfluent.field_data")
 
+_to_field_name_str = naming_strategy().to_string if naming_strategy else lambda s: s
```

Comment:
👍

Reply:
shall we import this from field data interfaces?

Reply:
I was curious about that. I di not foresee a dependency on `ansys.fluent.core` here. I will follow up with that change.